src/main/resources/templates/db_scripts/schema.json

```diff
@@ -180,6 +180,12 @@
           "caseSensitive": false,
           "removeAccents": false
         },
+        {
+          "fieldName": "requestType",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": false
```

Comment:
Are these the settings that all b-tree indexes used for `==` should use now?

Reply:
RMB has been changed to always use the accent and case setting of the underlying index.
The module can use the setting matching the front-end user needs.